### ISSUE TITLE
Google Drive 'open with Photopea' issue

### ISSUE THREAD

**Issue by tuconeves** [photopea/photopea#3815]
[https://streamable.com/4o9zwc](url)

Here is a screencapture reproducing the problem. Im using Firefox on Ubuntu, but the same happens in Windows running Chrome and Edge-Chromium. (No extensions installed)

After opening files directly from Gdrive, Photopea shows 'loading files from google drive' but anything happens after that. Any file format. Opening files from internal storage, works fine. Same google account is logged in Photopea

**Comment by photopea** [comment]
Here is how it works in Firefox: https://www.youtube.com/watch?v=p7kjmwhlNrA

**Comment by photopea** [comment]
Are you using any browser extensions? Try these steps: https://www.photopea.com/tuts/photopea-doesnt-work-what-to-do/

Yes, we are planning to add a file browsing mechanism for google drive soon :)

**Comment by Randmaster123** [comment]
i have the same problem... i use extensions and i deactivate them, but the image never loads... even i've tried in my smartphone and nothing change.
https://drive.google.com/file/d/1A84hYR2sJKCJH43YvhosGxwubYRkzZET/view?usp=sharing
(My english is bad, i'm speak spanish)

**Comment by photopea** [comment]
@Randmaster123 there should be a popup, where you grant Photopea the access to your GDrive. Look at the top of your browser, there should be a message about your browser blocking a popup window.

**Comment by tuconeves** [comment]
Hi! Sorry not replying earlier

Playing with the extensions menu I have found that Photopea has a problem with **any** extension used to create a different New Tab.
I do use one of them in Chrome to be redirected to Chrome://apps every time I open a new tab. I have disabled it to make those tests but I have noticed that Photopea only works again after browser restarts.
But there were no extensions on Firefox and MS Edge.

This problem seems to appears every time there are a extension installed. But it also appears (very frequently) in fresh installations (like the one in my first post). I could not find a solution than download the file and editting it locally.

:) happy to know you are planing to implement Gdrive file chooser! it i'll be a life saver.

Thanks!





> i have the same problem... i use extensions and i deactivate them, but the image never loads... even i've tried in my smartphone and nothing change.
@Randmaster123 I have noticed that sometimes this message "loading from google drive" keeps on the screen for a long time and it means it is waiting for some permissions, try check if your google account havent blocked Photopea to acess your files accounts.google.com
